tutorcelery/patches/k8s-override

```diff
@@ -0,0 +1,22 @@
+{% for service in ["lms", "cms"] %}
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{service}}-worker
+spec:
+  template:
+    spec:
+      containers:
+        - name: {{service}}-worker
+          image: {{ DOCKER_IMAGE_OPENEDX }}
+          args:
+            - "celery"
+            - "--app={{service}}.celery"
+            - "worker"
+            - "--loglevel=info"
+            - "--hostname=edx.{{service}}.core.default"
+            - "--max-tasks-per-child=100"
+            - "--queues=edx.{{service}}.core.default"
```

Comment:
`queue` or `queues`?

Reply:
Apparently is [queues](https://docs.celeryq.dev/en/stable/reference/cli.html#cmdoption-celery-worker-Q)